monster.py

```diff
@@ -40,21 +40,24 @@ def think(self):
 
 
 class Hunter(Monster):
-    def __init__(self, room, x=0, y=0):
+    def __init__(self, room, x=0, y=0, speed=0.15):
```

Comment:
ok suhlasim,
zajtra to fixnem nebude to parameter iba tvrdy bulhar v konstruktore inac ostane zmeny moze byt?